src/Linux/Avalonia.LinuxFramebuffer/Output/DrmOutput.cs

```diff
@@ -144,7 +164,9 @@ uint GetCrtc()
 
             using (_deferredContext.MakeCurrent(_eglSurface))
             {
-                _deferredContext.GlInterface.ClearColor(0, 0, 0, 0);
+                _deferredContext.GlInterface.ClearColor(_outputOptions.InitialBufferSwappingColor.R,
```

Comment:
I think [glClearColor](https://docs.gl/es2/glClearColor) accepts values in 0.0-1.0 range rather than 0-255 range.

Reply:
Oh yeah you are right.
Should we convert it or change DrmOutputOptions?

Reply:
Convert